mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -303,8 +303,10 @@ def LLVM_GEPOp : LLVM_Op<"getelementptr", [Pure,
     as indices. In the case of indexing within a structure, it is required to
     either use constant indices directly, or supply a constant SSA value.
 
-    An optional 'inbounds' attribute specifies the low-level pointer arithmetic
+    Optional attributes can be used to specify the low-level pointer arithmetic
     overflow behavior that LLVM uses after lowering the operation to LLVM IR.
+    The acceptable attributes could be one of or the combination of 'inbounds',
+    'nusw' or 'nuw'.
```

Comment:
Could it make sense to provide a very short definition of the flags (I may be off in terms of the actual semantics):

Valid options include 'inbounds' (pointer arithmetic must be within object bounds),
'nsw' (no signed wrap), and 'nuw' (no unsigned wrap). The flags can be set individually or in combination.

Reply:
I'll not it's `nusw` - no unsigned signed wrap . And the separate note that `inbounds` implies `nusw`, which the enum definition ensures

Reply:
@krzysz00 thanks for the correction! I did edit the text above. Hopefully it is correct now.

Reply:
How is it ensured by the enum definition? I would expect that this means that the enum is a bitfield and the `inbounds` has the bit set for both positions?

Reply:
Ah I misread: this is actually done (I had only seen `GEPInboundsFlag` instead of `GEPInbounds` which does the grouping).

Reply:
What happens if someone uses GEPInboundsFlag to build the operation?

Reply:
I think we need a verifier somewhere (probably on the property or on the op) to catch inbounds_flag without nusw 

Currently, I think that combination goes to an LLVM assertion in the translator

Reply:
If `inbounds_flag` is used the assertion will be triggered in LLVM: https://github.com/llvm/llvm-project/blob/main/llvm/include/llvm/IR/GEPNoWrapFlags.h#L35

Reply:
Yeah I would also be in favor of a verifier in MLIR itself. Triggering an assert after exporting to LLVM IR is not ideal.